### DISCUSSION TITLE
3.0版本开始，元信息加密后，连手动输入链接都无法下载了。

### DISCUSSION THREAD

**Discussion by fzlychee2019** [AlistGo/alist#2553] [General]
原来2.3版本在元信息加密后，只是目录隐藏不让用户浏览，但是手动输入地址还可以下载文件，但是3.0版本后，直到最新的3.5.1版，只要在元信息加密后，手动输入也一样无法下载了，这非常不方便，希望能恢复到2.3版的允许手动输入地址下载文件。

**Comment by iemobiu** [comment]
借您这里，问下，是手机浏览器无法播放挂载的网盘里的视频吗？谢谢

**Reply by wenge110110** [reply]
我也是这个情况 手机无法观看 电脑可以
